toolkit/scripts/toolchain.mk

```diff
@@ -297,3 +305,23 @@ $(toolchain_rpms): $(TOOLCHAIN_MANIFEST) $(depend_REBUILD_TOOLCHAIN)
 		$(call print_error,\nToolchain download failed. See above errors for more details.) \
 	}
 endif
+
+# ./out/RPMS is reserved for RPMs generated by the tooling, all other RPMs are cached in the ./build folder. If REBUILD_TOOLCHAIN=y is set
+# the toolchain RPMs should also be placed into out/RPMs.
+
+# ./out/SRPMS are still handled by build_official_toolchain_rpms.sh as a side effect of building the toolchain tar.gz.
+ifeq ($(REBUILD_TOOLCHAIN),y)
+$(RPMS_DIR): $(toolchain_out_rpms)
+
+# For each toolchain rpm in ./out/RPMS, add a dependency on its counterpart in the normal toolchain directory
+$(foreach toolchain_out_rpm,$(toolchain_out_rpms),$(eval $(toolchain_out_rpm): $(filter %/$(notdir $(toolchain_out_rpm)),$(toolchain_rpms))))
+$(toolchain_out_rpms): ; $(warning $^)
```

Comment:
Why print a warning for all toolchain out RPMs? It seems noisy and confusing. Or am I getting the `$(warning $^)` part wrong?

Reply:
Don't recall the exact reason for it (its been a while...), I think we can drop it though. There are some weird syntax with adding $(warnings...) to the makefile. I'll just pull the warning, best I can tell it's just extra debug code.